Modules/ModParser-3_0.lua

```diff
@@ -742,6 +742,7 @@ local modTagList = {
 	["per (%d+)%% lightning resistance above 75%%"] = function(num) return { tag  = { type = "PerStat", stat = "LightningResistOver75", div = num } } end,
 	["per totem"] = { tag = { type = "PerStat", stat = "ActiveTotemLimit" } },
 	["per summoned totem"] = { tag = { type = "PerStat", stat = "ActiveTotemLimit" } },
+	["for each summoned totem"]{ tag = { type = "PerStat", stat = "ActiveTotemLimit" } },
```

Comment:
Forgot to add the ` = ` between the `totem"]{ tag = `

Reply:
Thanks, it's amended now.